### PR TITLE
Update Gems

### PR DESCRIPTION
**Pull request by bowerman**

### CODE REVIEW

**Comment by BM5k** [comment]
@bowerman I’ve fixed the CI build on this. You mind if I push to this branch?

**Comment by bowerman** [comment]
doooit

**Comment by BM5k** [comment]
So, there is a problem with the latest released version of rubocop. Specifically, one of the new cops in that version, `Performance/Count` errors while looking at `app/models/play.rb`. The issue has been resolved in the master branch of rubocop, but the fix hasn’t been released.

Do we want to (a) commit to using the master branch for rubocop or (b) disable the broken cop?

Once we decide, we can squash & merge.

File: .devfu-rubocop.yml [hunk 1]
```diff
@@ -107,11 +107,17 @@ Style/SingleLineBlockParams:
 Style/SingleSpaceBeforeFirstArg:
   Enabled: false
 
+Style/SpaceAroundOperators:
+  Enabled: false
```
Comment (bowerman) [inline .devfu-rubocop.yml]: want comments around these to remind us to turn them back on?

**Comment by bowerman** [comment]
re-staged and all up to date / squashed.

**Pull request merged**